comms/src/connection/peer_connection/listener.rs

```diff
@@ -557,6 +535,37 @@ impl PeerConnectionListener {
         payload
     }
 
+    fn send_to_sink(&mut self, mut payload: FrameSet) -> Result<(), PeerConnectionError> {
```

Comment:
This could maybe be DRYed up with the one in the Dailer.

Reply:
Agreed, it's basically laziness because these connections are going away :)

Reply:
Then by all means.